nbbuild/licenses/CC0-markdown

```diff
@@ -0,0 +1,35 @@
+DEDICATED TO THE PUBLIC DOMAIN BY DUSTIN CURTIS
+
+To the extent possible under law, Dustin Curtis has waived all copyright and related or neighboring rights to Markdown Mark.
+
+The Markdown Mark has been dedicated to the public domain. It is protected by the Creative Commons CC0 Universal Public Domain Dedication license. You can read the entire license below or at https://creativecommons.org/publicdomain/zero/1.0/deed.en.
+
+CC0 UNIVERSAL PUBLIC DOMAIN DEDICATION LICENSE
```

Comment:
Isn't this identical to `CC0-v10`? From my understanding of CC0, the author releases all copyrights/restricts he/she could pose upon a work. For the npm the icon is a recolored version of a CC0 basis - that derived file was marked as ALv2 and the header gives the history.

Reply:
I think we can use CC0-v10 - I did that in the latest update (there are some formatting changes, and the initial caption is different).

Thanks for the comment!